### DISCUSSION TITLE
Can we use Security Onion 2 more like a SIEM than a NSM ?

### DISCUSSION THREAD

**Discussion by NRGLine4Sec** [Security-Onion-Solutions/securityonion#1629] [General]
Can we send log to SO with [Fluentd](https://www.fluentd.org/) for exemple, or [vector](https://vector.dev/) and get the full other features provides by SO (Alerts, TheHive, Playbook ...) without NIDS capabilities (Suricata/Zeek)?

**Comment by dougburks** [comment]
The short answer is that Security Onion is open source and so anything is technically possible depending on how much time you want to invest in it! :smile:
